### PR TITLE
Fix uninitialized Rocksdb.level0_stop_writes_trigger

### PR DESCRIPTION
**Pull request by ShooterIT**

### CODE REVIEW

**Review by karelrooted — DISMISSED** [review]

**Review by git-hulk — DISMISSED** [review]

**Comment by ShooterIT** [comment]
Hi, i add the support to config level0_stop_writes_trigger, because we have this variable in `config.h`, moreover, we shouldn't  bind them together, they are changed respectively for tuning.

File: src/config.cc [hunk 1]
```diff
@@ -288,6 +282,8 @@ void Config::initFieldCallback() {
       {"rocksdb.max_background_compactions", set_db_option_cb},
       {"rocksdb.max_background_flushes", set_db_option_cb},
       {"rocksdb.compaction_readahead_size", set_db_option_cb},
+      {"rocksdb.level0_slowdown_writes_trigger", set_db_option_cb},
```
Comment (karelrooted) [inline src/config.cc]: level0_slowdown_writes_trigger and level0_stop_writes_trigger is a column family option, can not use setdboption

**Review by karelrooted — CHANGES_REQUESTED** [review]

File: src/config.cc (hunk 1 above)
Comment (ShooterIT) [reply src/config.cc]: oh, thanks, i found unit test failed

**Review by karelrooted — APPROVED** [review]

**Review by git-hulk — APPROVED** [review]

**Comment by ShooterIT** [comment]
@Mergifyio backport 1.3

**Pull request merged**